Yona/Yona/Info.plist

```diff
@@ -2,6 +2,20 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <key>Fabric</key>
+    <dict>
+        <key>APIKey</key>
+        <string>1b30b6855885b6c0d1b05b34efcf5d6cbe2a5f13</string>
```

Comment:
This key should not be stored here. See https://stackoverflow.com/questions/36988414/is-it-safe-to-expose-your-fabric-api-key

Reply:
Ok. I Will look for alternative way and update the code.

Reply:
I am not entirely sure if this StackOverflow question can be trusted. Here is some information that is more detailed: https://www.herzbube.ch/blog/2016/08/how-hide-fabric-api-key-and-build-secret-open-source-project